deploy/bav-spec.yaml

```diff
@@ -625,5 +733,14 @@ components:
       in: header
       description: A UUID generated by the Session API to act as a primary key for the Session in DynamoDB
       required: true
+      schema:
+        type: string
+    Authorization:
+      in: header
+      name: Authorization
+      example: bearer abcdefghijklmnopqrstuvwxyz
+      required: true
+      description: >-
+        OAuth Bearer token - this it the `authorizationCode` returned by `POST /claimedIdentity`
```

Comment:
Is the endpoint going to be called `/claimedIdentity`?

Reply:
Updated